content/rosa/ip-addressing-and-subnets/index.md

```diff
@@ -0,0 +1,101 @@
+
+For most users, the Machine CIDR range should be unique across all environments and be assigned out of some kind of IPAM scheme.
+
+It's also notable that while the ROSA installer requires the *specification* of the Machine CIDR range at install time, the cluster itself does nothing to configure the *assignment* or *routing* associated with that range, that is all handled by the underlying VPC resources on AWS.
+
+Typically, the Machine CIDR range will either be the same as or a subset of the CIDR range for the VPC the cluster is being built in. 
+
+## Subnetting
+
+Because AWS subnets cannot span more than a single AZ, and subnetting is based on splitting up larger networks into multiples of the number '2', a lot of the habits formed around managing networks in a datacenter environment can lead to poor utilization of the IP address space.
+
+Let's start with an example. ACME industries (has to be ACME right?), wants to build a ROSA cluster in the AWS `us-east-1` region, and they want it to be HA. The team that will operate the cluster goes to their network engineering group and requests a CIDR block for this use as dictated by ACME policy and best practice. The network engineering group assigns the block `10.0.0.0/22` in their IPAM system and the team goes forth to provision AWS resources.
+
+ROSA clusters need to reside in a VPC with both public and private subnets, so the conventional wisdom for an HA config would dictate this VPC contain at least six subnets, three private and three public, which would traditionally be accomplished by splitting the assigned CIDR range up like so:
+
+|Subnet address	|Netmask	        |Addresses	                |Useable IPs	                |Hosts  |
+|---------------|-------------------|---------------------------|-------------------------------|-------|
+|10.0.0.0/25	|255.255.255.128	|10.0.0.0 - 10.0.0.127	    |10.0.0.1 - 10.0.0.126	        |126	|
+|10.0.0.128/25	|255.255.255.128	|10.0.0.128 - 10.0.0.255	|10.0.0.129 - 10.0.0.254	    |126	|
+|10.0.1.0/25	|255.255.255.128	|10.0.1.0 - 10.0.1.127	    |10.0.1.1 - 10.0.1.126	        |126	|
+|10.0.1.128/25	|255.255.255.128	|10.0.1.128 - 10.0.1.255	|10.0.1.129 - 10.0.1.254	    |126	|
+|10.0.2.0/25	|255.255.255.128	|10.0.2.0 - 10.0.2.127	    |10.0.2.1 - 10.0.2.126	        |126	|
+|10.0.2.128/25	|255.255.255.128	|10.0.2.128 - 10.0.2.255	|10.0.2.129 - 10.0.2.254	    |126	|
+|10.0.3.0/25	|255.255.255.128	|10.0.3.0 - 10.0.3.127	    |10.0.3.1 - 10.0.3.126	        |126	|
+|10.0.3.128/25	|255.255.255.128	|10.0.3.128 - 10.0.3.255	|10.0.3.129 - 10.0.3.254	    |126	|
+
+This presents a couple of issues. 
+
+First, there are only six subnets required by this ROSA cluster. This cluster doesn't rely on any AWS-native services that require their own subnets (e.g., RDS), so the `10.0.3.0/24` block (the last two lines) would go un-utilized.
```

Comment:
Is there certain CIDRs beyond /24 that are specifically friendlier with a 3 or 6 subnet split ?